src/plan_bot.py

```diff
@@ -149,6 +168,9 @@ def reply(post, reply_string: str, send=False, simulate=False):
     :return: did_reply – whether an actual or simulated reply was made
     """
 
+    if parent and hasattr(post, 'parent'):
+        post = standardize(post.parent())
```

Comment:
I assume that if a post has no parent (I.e. if it is a submission), that this doesn't crash? 

We should just be sure to test this on dev :)

Reply:
The `hasattr` should catch that. But yes, let’s test on dev. I’ll probably need a hand there with the api keys and whatnot.

Reply:
Ok cool! I think for now, I’d like to avoid passing around API keys and handle deployments until/unless that becomes a bottleneck. (And/or we can set up Travis to do deployments)

Once this merges to master, I’ll click the button and we can test it on WPBSandbox